lib/ts/nextjs.ts

```diff
@@ -230,6 +230,24 @@ export default class NextJS {
         return result;
     }
 
+    static async getInitialSessionAuthContext(session: SessionContainer | undefined): Promise<SSRSessionContextType> {
```

Comment:
I don't think we need this function. And if we are going to keep it, why is it in the nextjs file?